modules/API/constants.js

```diff
@@ -15,3 +15,18 @@ export const API_ID = parseURLParams(window.location).jitsi_meet_external_api_id
  * The payload name for the datachannel/endpoint text message event.
  */
 export const ENDPOINT_TEXT_MESSAGE_NAME = 'endpoint-text-message';
+
+/**
+ * The interval for the assummed bandwidth bps value.
+ */
+export const ASSUMED_BANDWIDTH_BPS = {
+    /**
+     * The max count of bps that can be set for the assumed bandwidth.
+     */
+    MAX_VALUE: 10000000,
```

Comment:
Does this come from the bridge?

Reply:
Yes, that's the limit we settled on - 10 Mbps.

Reply:
That's configurable, one could set it to 100 Mbps.